Tester.java

```diff
@@ -8,10 +8,429 @@
+                Pass ID Number: 1008
+                Luxury Rating: 6
+                Credits: 30
+                Journey Points: 0
+                ******************** .Actual:
+                """+ "\n" + Wayward.getAllPassesOnAllIslands() + "\n");
+        //        Testing option 3 List all passes on one Island
+        System.out.println("Option 3, List all passes on one Island in this case, island name: (Base)");
+        String islandName = enter.next();
+        System.out.println(
+                """
+                Expected:
+                ********************
+                Guest Name: Lynn
+                Pass ID Number: 1000
+                Luxury Rating: 5
+                Credits: 10
+                Journey Points: 0
+                ********************
+                ********************
+                Guest Name: May
+                Pass ID Number: 1001
+                Luxury Rating: 3
+                Credits: 30
+                Journey Points: 0
+                ********************
+                ********************
+                Guest Name: Nils
+                Pass ID Number: 1002
+                Luxury Rating: 10
+                Credits: 0
+                Journey Points: 0
+                ********************
+                ********************
+                Guest Name: Olek
+                Pass ID Number: 1003
+                Luxury Rating: 1
+                Credits: 12
+                Journey Points: 0
+                ********************
+                ********************
+                Guest Name: Pan
+                Pass ID Number: 1004
+                Luxury Rating: 3
+                Credits: 3
+                Journey Points: 0
+                ********************
+                ********************
+                Guest Name: Quin
+                Pass ID Number: 1005
+                Luxury Rating: 1
+                Credits: 30
+                Journey Points: 0
+                ********************
+                ********************
+                Guest Name: Raj
+                Pass ID Number: 1006
+                Luxury Rating: 4
+                Credits: 5
+                Journey Points: 0
+                ********************
+                ********************
+                Guest Name: Sol
+                Pass ID Number: 1007
+                Luxury Rating: 7
+                Credits: 20
+                Journey Points: 0
+                ********************
+                ********************
+                Guest Name: Tel
+                Pass ID Number: 1008
+                Luxury Rating: 6
+                Credits: 30
+                Journey Points: 0
+                ********************
+                .Actual:""" + "\n" + Wayward.getAllPassesOnIsland(islandName) + "\n");
+        //        Testing option 4-> Find location of a pass
+        System.out.println("(True version option 4) Enter pass id:(1006)");
```

Comment:
Redo everything from this point onwards. There should be no inputting in this class